language/move-vm/types/src/data_store.rs

```diff
@@ -19,27 +19,12 @@ pub trait DataStore {
     // StateStore operations
     // ---
 
-    /// Publish a resource.
-    fn publish_resource(
-        &mut self,
-        addr: AccountAddress,
-        ty: Type,
-        gv: GlobalValue,
-    ) -> PartialVMResult<()>;
-
-    /// Get a reference to a resource.
-    fn borrow_resource(
-        &mut self,
-        addr: AccountAddress,
-        ty: &Type,
-    ) -> PartialVMResult<Option<&GlobalValue>>;
-
-    /// Transfer ownership of a resource to the VM.
-    fn move_resource_from(
+    /// TBA
```

Comment:
Comment needed?

Reply:
Ah good catch. Will fix it in the base commit.